microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -485,4 +485,13 @@ Issued: 2024-04-08
 
 {product-title} release 4.14.20 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1682[RHBA-2024:1682] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1681[RHSA-2024:1681] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-14-21-dp"]
+=== RHBA-2024:1767 - {microshift-short} 4.14.21 bug fix and enhancement advisory
+
+Issued: 2024-04-18
```

Comment:
OK that date is in the future?

Reply:
Yes, that's the ART date for the upcoming z-stream.